HotelManagerment/com.spring.sample/src/com/spring/sample/controller/RoomCategoryController.java

```diff
@@ -0,0 +1,19 @@
+package com.spring.sample.controller;
+import java.util.Locale;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@Controller
+@EnableWebMvc
+public class RoomCategoryController {
+	private static final Logger logger = LoggerFactory.getLogger(RoomCategoryController.class);
+	@GetMapping(value = "/room-category")
```

Comment:
```suggestion
	@GetMapping(value = "/room-categories/index")
```
Nó như ri nhé
em có thể tìm hiểu về chuẩn đặt url theo restful